src/main/java/application/controllers/TreeController.java

```diff
@@ -131,14 +130,15 @@ public void colorSelectedStrains() {
      */
     public void applyCellHighlight(Cell cell) {
         if (cell instanceof LeafCell) {
-            List<Cell> parentList = new ArrayList<>();
-            parentList.add(cell);
+            String temp = ((LeafCell) cell).getName();
             collectedStrains.clear();
-            collectedStrains.add(cell);
 
-            applyColorUpwards(parentList,
-                    determineLinColor(metaData.get(((LeafCell) cell).getName())),
-                    4.0);
+            if (temp.contains("TKK")) {
```

Comment:
There's no MetaData for those strains, their information might be redundant and I cannot assign a random color because of the strict color scheme for Lineages. I will fix this by highlighting them with YELLOW. But beware, this might be incorrect and it should be addressed at the next upcoming customer meeting.